src/models/databases_connectors/databases_access.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+interface DatabasesAccess
+{
+	public function isExcerciceExist(int $id): bool;
+
+	public function createExercice(string $title): int;
+
+	public function getExcerciceTitleFromId(int $id): string;
```

Comment:
Why make the function name to precise the parameter, when the parameter has a clear name

change it fot getExerciceTitle(int $id)

In future cases we could add function overload with other parameter type